src/services/discord/loggers/memberUpdateLogger.js

```diff
@@ -0,0 +1,39 @@
+const { MessageEmbed } = require('discord.js')
+const { main_server, images, channels } = require('../info')
+
+module.exports = (oldMember, newMember) => {
+  if (newMember.guild.id === main_server) {
```

Comment:
Usually this many if/for statements become more readable by refactoring into functions or by using `&&` to collapse multiple `if` statements. You can also do logic gates, which we can talk about when we meet next.